hydra-auction.cabal

```diff
@@ -22,6 +22,7 @@ data-files:
   data/credentials/grace.vk
   data/credentials/hans.sk
   data/credentials/hans.vk
+  data/hydra-keys/*
```

Comment:
I don't think you can use the `*` here. iirc you need to pass the fully qualified paths

Reply:
I checked, docs say you can: https://cabal.readthedocs.io/en/3.4/cabal-package.html#pkg-field-data-files

Also it is covered by test.